js/classes/animator.js

```diff
@@ -45,7 +45,7 @@ class Animator {
         }
         
         if (this.elapsedTime > this.totalTime && this.looped) this.elapsedTime -= this.totalTime;
-        const frame = this.currentFrame();
+        const frame = this.currentFrame() + ((this.reversed) ? 1 : 0);
```

Comment:
Nice work. So we needed to add one to fix it? Wondering why. Does the reverse starting index equal the amount of frames indicating it should end at 1 instead of 0?